### DISCUSSION TITLE
MVP I

### DISCUSSION THREAD

**Discussion by bellomuboye** [bellomuboye/make-we-yarn#3] [General]
Set up React and link to Firebase #1
Set up Styling, Linting, Testing, CI/CD
Authentication with email + password
Authentication with Google, Twitter
Mail confirmation after successful signup
Single Chat Room
Basic Profile Update
List of members in Room
Direct Messaging

See full bucket here #2

**Comment by fmoliveira** [comment]
Sounds great!

**Reply by fmoliveira** [reply]
And very good to create a milestone for these tasks!
